system/netdata.service.in

```diff
@@ -71,6 +71,10 @@ ProtectControlGroups=on
 ReadWriteDirectories=/run/netdata
 # This is needed to make email-based alert deliver work if Postfix is the email provider on the system.
 ReadWriteDirectories=-/var/spool/postfix/maildrop
+# LXCFS directories (https://github.com/lxc/lxcfs#lxcfs)
+# If we don't set them explicitly, systemd mounts procfs from the host. See https://github.com/netdata/netdata/issues/14238. 
+BindReadOnlyPaths=-/proc/cpuinfo -/proc/diskstats -/proc/loadavg -/proc/meminfo
+BindReadOnlyPaths=-/proc/stat -/proc/swaps -/proc/uptime -/proc/slabinfo
```

Comment:
An alternative is the `BindReadOnlyPaths=/proc:/proc:rbind` one-liner (suggested in #14238). It works, but I decided to not use it because it makes `/proc/mounts` kinda messy.

<details>
<summary>proc rbind</summary>

```cmd
rpool/data/subvol-111-disk-0 / zfs rw,noatime,xattr,posixacl 0 0
none /dev tmpfs rw,relatime,size=492k,mode=755,uid=100000,gid=100000,inode64 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
proc /dev/.lxc/proc proc rw,relatime 0 0
sys /dev/.lxc/sys sysfs rw,relatime 0 0
udev /dev/full devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/null devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/random devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/tty devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/urandom devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/zero devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/ptmx devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/console devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/tty1 devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/tty2 devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev,uid=100000,gid=100000,inode64 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
proc /proc/sys proc ro,relatime 0 0
proc /proc/sys/net proc rw,nosuid,nodev,noexec,relatime 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,nosuid,nodev,noexec,relatime 0 0
none /proc/sys/kernel/random/boot_id tmpfs ro,nosuid,nodev,noexec,relatime,size=492k,mode=755,uid=100000,gid=100000,inode64 0 0
proc /proc/sysrq-trigger proc ro,relatime 0 0
lxcfs /proc/cpuinfo fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/diskstats fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/loadavg fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/meminfo fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/stat fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/swaps fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/uptime fuse.lxcfs rw,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
sysfs /sys sysfs ro,nosuid,nodev,noexec,relatime 0 0
sysfs /sys/devices/virtual/net sysfs rw,nosuid,nodev,noexec,relatime 0 0
fusectl /sys/fs/fuse/connections fusectl rw,nosuid,nodev,noexec,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,nosuid,nodev,noexec,relatime 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
efivarfs /sys/firmware/efi/efivars efivarfs rw,nosuid,nodev,noexec,relatime 0 0
none /sys/fs/cgroup cgroup2 ro,nosuid,nodev,noexec,relatime 0 0
lxcfs /sys/devices/system/cpu/online fuse.lxcfs rw,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
tmpfs /run tmpfs rw,nosuid,nodev,size=105641444k,nr_inodes=819200,mode=755,uid=100000,gid=100000,inode64 0 0
tmpfs /run/lock tmpfs rw,nosuid,nodev,noexec,relatime,size=5120k,uid=100000,gid=100000,inode64 0 0
proc /proc proc ro,nosuid,nodev,noexec,relatime 0 0
proc /proc/sys proc ro,noexec,relatime 0 0
proc /proc/sys/net proc ro,nosuid,nodev,noexec,relatime 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc ro,nosuid,nodev,noexec,relatime 0 0
none /proc/sys/kernel/random/boot_id tmpfs ro,nosuid,nodev,noexec,relatime,size=492k,mode=755,uid=100000,gid=100000,inode64 0 0
proc /proc/sysrq-trigger proc ro,noexec,relatime 0 0
lxcfs /proc/cpuinfo fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/diskstats fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/loadavg fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/meminfo fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/stat fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/swaps fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/uptime fuse.lxcfs ro,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
tmpfs /run/credentials tmpfs ro,nosuid,nodev,noexec,size=105641444k,nr_inodes=819200,mode=755,uid=100000,gid=100000,inode64 0 0
tmpfs /run/systemd/incoming tmpfs ro,nosuid,nodev,size=105641444k,nr_inodes=819200,mode=755,uid=100000,gid=100000,inode64 0 0
```
</details>

<details>
<summary>current approach</summary>

```cmd
rpool/data/subvol-111-disk-0 / zfs rw,noatime,xattr,posixacl 0 0
none /dev tmpfs rw,relatime,size=492k,mode=755,uid=100000,gid=100000,inode64 0 0
mqueue /dev/mqueue mqueue rw,relatime 0 0
proc /dev/.lxc/proc proc rw,relatime 0 0
sys /dev/.lxc/sys sysfs rw,relatime 0 0
udev /dev/full devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/null devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/random devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/tty devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/urandom devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
udev /dev/zero devtmpfs rw,nosuid,relatime,size=264070148k,nr_inodes=66017537,mode=755,inode64 0 0
devpts /dev/pts devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/ptmx devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/console devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/tty1 devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
devpts /dev/tty2 devpts rw,nosuid,noexec,relatime,gid=100005,mode=620,ptmxmode=666,max=1026 0 0
tmpfs /dev/shm tmpfs rw,nosuid,nodev,uid=100000,gid=100000,inode64 0 0
sysfs /sys sysfs ro,nosuid,nodev,noexec,relatime 0 0
sysfs /sys/devices/virtual/net sysfs rw,nosuid,nodev,noexec,relatime 0 0
fusectl /sys/fs/fuse/connections fusectl rw,nosuid,nodev,noexec,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,nosuid,nodev,noexec,relatime 0 0
securityfs /sys/kernel/security securityfs rw,nosuid,nodev,noexec,relatime 0 0
pstore /sys/fs/pstore pstore rw,nosuid,nodev,noexec,relatime 0 0
efivarfs /sys/firmware/efi/efivars efivarfs rw,nosuid,nodev,noexec,relatime 0 0
none /sys/fs/cgroup cgroup2 ro,nosuid,nodev,noexec,relatime 0 0
lxcfs /sys/devices/system/cpu/online fuse.lxcfs rw,nosuid,nodev,noexec,relatime,user_id=0,group_id=0,allow_other 0 0
tmpfs /run tmpfs rw,nosuid,nodev,size=105641444k,nr_inodes=819200,mode=755,uid=100000,gid=100000,inode64 0 0
tmpfs /run/lock tmpfs rw,nosuid,nodev,noexec,relatime,size=5120k,uid=100000,gid=100000,inode64 0 0
tmpfs /run/user/1000 tmpfs rw,nosuid,nodev,relatime,size=52820720k,nr_inodes=13205180,mode=700,uid=101000,gid=101000,inode64 0 0
proc /proc proc rw,nosuid,nodev,noexec,relatime 0 0
lxcfs /proc/cpuinfo fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/diskstats fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/loadavg fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/meminfo fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
proc /proc/slabinfo proc ro,nosuid,nodev,noexec,relatime 0 0
lxcfs /proc/stat fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/swaps fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
lxcfs /proc/uptime fuse.lxcfs ro,nosuid,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
tmpfs /run/credentials tmpfs ro,nosuid,nodev,noexec,size=105641444k,nr_inodes=819200,mode=755,uid=100000,gid=100000,inode64 0 0
tmpfs /run/systemd/incoming tmpfs ro,nosuid,nodev,size=105641444k,nr_inodes=819200,mode=755,uid=100000,gid=100000,inode64 0 0
```

</details>